### ISSUE TITLE
Support for Header and Footer UI Banners

### ISSUE THREAD

**Issue by gaby** [openobserve/openobserve#1056]
### Which OpenObserve functionalities are relevant/related to the feature request?

other

### Description

Add support for Header and Footers UI banners. These are normally used in Enterprise/Gov settings. They are also used to display: Events, Alerts, Messages to users. For ex: "NOTICED: System will be upgraded at XX:YY with a downtime of 1hr".

Ideally these should be configurable from the settings/admin account, including choosing the RGB color and text. The banners should always show up, even if the user is not logged in, or maybe that's configurable with a Public checkmark option?

### Proposed solution

Redhat styling patterns:
- https://www.patternfly.org/v4/components/banner/react-demos/#top-and-bottom-banner
- https://www.patternfly.org/v3/pattern-library/communication/classification-banner/index.html

### Alternatives considered

N/a

**Comment by gaby** [comment]
Updated Reference URL: https://www.patternfly.org/components/banner/design-guidelines/

**Comment by hengfeiyang** [comment]
@prabhatsharma @bjp232004 i think this is a good feature.

**Comment by prabhatsharma** [comment]
> [@prabhatsharma](https://github.com/prabhatsharma) [@bjp232004](https://github.com/bjp232004) i think this is a good feature.

Yes, We should move the config to the UI first from env variables.

**Comment by gaby** [comment]
@prabhatsharma Was the config ever moved to the UI ?

**Comment by hengfeiyang** [comment]
a few config can change in the UI. haven't done all the setting UI.

**Comment by gaby** [comment]
@hengfeiyang Thanks, looking forward
